spec/fixtures/golden.md

```diff
@@ -1,12 +1,13 @@
 # Entities
 
-## Mock::Bar::Entity
+## Mock::Bar
```

Comment:
can we replace all `::` with just `:` - it makes more sense for the doc + makes better anchors

Reply:
I don't like it at all, but OK.